assets/scss/_variables_project.scss

```
@@ -34,14 +34,14 @@ $otel-component-colors: (
   'utilities': #2cd3b4,      // Keep existing
 );
 
-$otel-registry-license-colors: (
+$in-toto-registry-license-colors: (
   'apache-20': #e83e8c,        // Keep existing
   'bsd-2-clause': #8ce83e,     // Keep existing
   'mit': #3e8ce8,              // Keep existing
   'artistic-10-perl': #e8c83e, // Keep existing
 );
 
-$primary: map-get($otel-colors, 'blue');       // Darker blue for primary elements
-$secondary: map-get($otel-colors, 'orange');   // Bright orange for secondary elements
-$secondary_1: map-get($otel-colors, 'java');   // Keep existing
+$primary: map-get($in-toto-colors, 'blue');       // Darker blue for primary elements
+$secondary: map-get($in-toto-colors, 'orange');   // Bright orange for secondary elements
+$secondary_1: map-get($in-toto-colors, 'java');   // Keep existing
```

```suggestion
$secondary: map-get($in-toto-colors, 'orange');   // Bright orange for secondary elements
```

We'll probably want to change the defintion of `$primary`, but we can do that later.